src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,25 @@
+package duke.task;
+
+/**
+ * The Todo, a task without any date or time attached to it.
+ */
```

Comment:
Maybe the head comments can be shrunk to one line?
This would make the code look more compact and neater if the header comment is not too long.
I noticed there were some quite short comments which could have been shorten to one line
`` /** The Todo, ... to it. */ ``

Reply:
Hmm I think all the comments in relation to javadocs is bcs I autogenerated javadocs for all elements and just edited it afterwards. I personally think that this looks nicer when I open the javadocs .html pages, maybe not in the .java files.